source/db/DBHandler.py

```diff
@@ -0,0 +1,168 @@
+import pprint
+from datetime import datetime
+
+from pymongo import MongoClient
+from bson import ObjectId
+import time
+
+
+class DBHandlerClass:
+    client = MongoClient()
+    db = client['Captains_Log_DB']
+
+    def __init__(self):
+        client = MongoClient()
+        self.db = client['Captains_Log_DB']
+
+    def insert_to_user_table(self, name, username, user_password):
```

Comment:
Don't use `_` for functions
Please use camel case or something alone the line of `insertUsertable`